docs/source/usage/install-gdal.rst

```diff
@@ -16,34 +16,72 @@ these permissions and more information about cluster permissions can be found
+    more on access modes `here <https://docs.databricks.com/en/compute/configure.html#access-modes>`__.
+
 GDAL Installation
 ####################
 
 Setup GDAL files and scripts
 ****************************
 Mosaic requires GDAL to be installed on the cluster. The easiest way to do this is to use the
-the mos.setup_gdal() function. This function will extract the GDAL files and scripts from the
-mosaic library and place them in the /dbfs/FileStore/geospatial/mosaic/gdal/ directory.
-This call is no longer needed in versions >= 0.3.12. The shared objects are now included in the
-databricks-mosaic-gdal pip installable bundle.
+the mos.setup_gdal() function.
+
+.. note::
+   (a) This is close in behavior to Mosaic < 0.4 series (prior to DBR 13), with new options
+       to pip install Mosaic for either ubuntugis gdal (3.4.3) or jammy default (3.4.1).
+   (b) `to_fuse_dir` can be one of `/Volumes/..`, `/Workspace/..`, `/dbfs/..`;
+        however, you should consider `setup_fuse_install()` for Volume based installs as that
+        exposes more options, to include copying JAR and JNI Shared Objects.
+
+.. function:: setup_gdal()
+
+    Generate an init script that will install GDAL native libraries on each worker node.
+    All of the listed parameters are optional. You can have even more control with setup_fuse_install function.
+
+    :param to_fuse_dir: Path to write out the init script for GDAL installation;
+                        default is '/Workspace/Shared/geospatial/mosaic/gdal/jammy'.
+    :type to_fuse_dir: str
+    :param with_mosaic_pip: Whether to configure a script that pip installs databricks-mosaic,
+                            fixed to the current version; default is False.
+    :type with_mosaic_pip: bool
+    :param with_ubuntugis: Whether to use ubuntugis ppa for GDAL instead of built-in;
+                           default is False.
+    :type with_ubuntugis: bool
+    :param script_out_name: name of the script to be written;
+                            default is 'mosaic-gdal-init.sh'.
+    :type script_out_name: str
+    :param override_mosaic_version: String value to use to override the mosaic version to install,
+                                    e.g. '==0.4.0' or '<0.5,>=0.4'; default is None.
+    :type override_mosaic_version: str
+    :rtype: bool
+
+    :example:
 
 .. code-block:: py
 
     import mosaic as mos
 
     mos.enable_mosaic(spark, dbutils)
-    mos.setup_gdal(spark)
-    GDAL setup complete.
-    Shared objects (*.so) stored in: /dbfs/FileStore/geospatial/mosaic/gdal/.
-    Init script stored in: /dbfs/FileStore/geospatial/mosaic/gdal/.
-    Please restart the cluster with the generated init script to complete the setup.
+    mos.setup_gdal()
+
+    +---------------------------------------------------------------------------------------------------------+
+    |::: Install setup complete :::                                                                           |
```

Comment:
@mjohns-databricks sphinx prefers if "|" is separated with " " on the side of text.

Reply:
good catch!